Gopkg.toml

```diff
@@ -37,7 +37,8 @@ required = [
 # Our master branch tracks knative/pkg master or a release.
 [[override]]
   name = "knative.dev/pkg"
-  branch = "master"
+  source = "github.com/pmorie/pkg"
```

Comment:
Change this back to knative.dev/pkg?

Reply:
I guess we can after https://github.com/knative/pkg/pull/1084 is in/merged

Reply:
Should we hold this until https://github.com/knative/pkg/pull/1084 is merged?